packages/familie-visittkort/src/index.tsx

```diff
@@ -11,13 +12,27 @@ export interface IProps {
     ident: string;
     kjønn: kjønnType;
     navn: string;
+    href?: string;
 }
 
-const Visittkort: React.StatelessComponent<IProps> = ({ alder, children, ident, kjønn, navn }) => {
+const Visittkort: React.StatelessComponent<IProps> = ({
+    alder,
+    children,
+    ident,
+    kjønn,
+    navn,
+    href,
+}) => {
```

Comment:
Kan en smudere variant være at man kan velge å sende inn custom komponent for navn istedet?

Reply:
Enig, oppdatert i 0390fe6